libs/sdk-ui-dashboard/src/model/commandHandlers/drill/crossFilteringHandler.ts

```diff
@@ -61,7 +61,7 @@ export function* crossFilteringHandler(ctx: DashboardContext, cmd: CrossFilterin
     const crossFilteringItemByWidget: ReturnType<ReturnType<typeof selectCrossFilteringItemByWidgetRef>> =
         yield select(selectCrossFilteringItemByWidgetRef(widgetRef));
 
-    const filtersCount = currentFilters.length;
+    const filtersCount = currentFilters.length + 1; //date filter is not included in currentFilters
```

Comment:
what about when there are multiple date filters?

Reply:
Others date filters are here, only the default one is missing ....